### PR TITLE
Added custom struct Operator as placeholder for CSV related fields

### PR DESCRIPTION
**Pull request by greyerof**
I brought back the old Operator struct from the old repo so all this
CSV-related information can be grouped under it.

### CODE REVIEW

**Comment by sebrandon1** [comment]
I would prefer if all of these funcs in `pkg/provider/provider.go` would have some sort of unit testing attached to them.

**Review by sebrandon1 — APPROVED** [review]

**Comment by sebrandon1** [comment]
Approved pending the Github Actions checks.

File: pkg/provider/provider.go [hunk 1]
```diff
@@ -67,12 +67,31 @@ type TestEnvironment struct { // rename this with testTarget
...
+type CsvInstallPlan struct {
+	// Operator's installPlan name
+	Name string `yaml:"name" json:"name"`
+	// BundleImage is the URL referencing the bundle image
+	BundleImage string `yaml:"bundleImage" json:"bundleImage"`
+	// IndexImage is the URL referencing the index image
+	IndexImage string `yaml:"indexImage" json:"indexImage"`
+}
+
+type Operator struct {
```
Comment (edcdavid) [inline pkg/provider/provider.go:85]: Create a .String() for Operator and use it in the code

**Review by edcdavid — APPROVED** [review]

**Comment by greyerof** [comment]
@sebrandon1 yes, I can add unit tests in further PR/s.

**Review by edcdavid — APPROVED** [review]

**Pull request merged**